src/index.js

```diff
@@ -17,5 +17,9 @@ addBtn.addEventListener('click', (e) => {
   warning.classList.add('hide');
   const newScore = new Scores(nameInput.value, scoreInput.value);
   newScore.add();
+});
```

Comment:
[OPTIONAL] Good job hitting the api :clap: , I think it will be better if we clear the form data once user submits the form with valid input, this will make sure the users do not submit same data multiple times by clicking the sublit button again and again and it will also acts as a confirmation that the data is submitted :+1:
here is a example of what I am talking about : 
```
const clearform = () => {
  document.getElementById('name').value = '';
  document.getElementById('score').value = '';
}
```
then we add the clearform() after adding the score

Reply:
Thank you very much